frontend/src/App.js

```diff
@@ -37,40 +41,46 @@ const App = () => {
   };
 
   return (
-    <Router>
-      <ErrorBoundary FallbackComponent={Error500page}>
-        <div className={classNames('app', `${currentTheme === 'dark' && 'dark-theme'}`)}>
-          <Header theme={currentTheme} toggleTheme={toggleTheme} />
-          <main>
-            <Switch>
-              <Route exact path="/" component={HomePage} />
-              <Route exact path="/:parameter(login|signup)" component={LoginSignupPage} />
-              <Route exact path="/profile/:username" component={ProfilePage} />
-              <Route exact path="/create-discussion" component={CreateDiscussionPage} />
-              <Route exact path="/create-article" component={CreateArticlePage} />
-              <Route exact path="/notifications" component={NotificationsPage} />
-              <Route exact path="/discussion/:slug" component={DiscussionPage} />
-              <Route exact path="/article/:slug" component={ArticlePage} />
-              <Route exact path="/search" component={SearchPage} />
-              <Route exact path="/settings">
-                <UserSettingsPage theme={currentTheme} toggleTheme={toggleTheme} />
-              </Route>
-              <Route path="/404" component={Error404page} />
-              <Redirect to="/404" />
-            </Switch>
-          </main>
-          <RestoreScroll />
-          <ToastContainer
-            position="bottom-right"
-            autoClose={3000}
-            hideProgressBar={false}
-            closeOnClick
-            draggable
-            pauseOnHover
-          />
-        </div>
-      </ErrorBoundary>
-    </Router>
+    <Provider store={store}>
```

Comment:
remove this; this is already in index.js wrapping the <App /> component.

Reply:
Yeah, I updated code.